front/src/applications/operationalStudies/components/MacroEditor/ngeToOsrd.ts

```
@@ -449,22 +451,59 @@ const handleCreateTimetableItem = async (
         }
       : undefined,
   };
-  const newTimetableItems = await dispatch(
-    osrdEditoastApi.endpoints.postTimetableByIdPacedTrains.initiate({
-      id: timetableId,
-      body: [pacedTrain],
-    })
-  ).unwrap();
-  if (newTimetableItems.length === 0) {
-    throw new Error('Failed to create paced train');
+  if (trainrun.direction === 'one_way') {
+    // This case should no longer happen in practice, as by default nge should create trains as round trips
```

Instead of keeping dead code, can we throw an error if this happens to be a one-way trainrun?

---

Eh, I don't like being too dependent on something that is nge config, and that could change in the future. But I suppose it would not be too hard to fix in that case.

---

We already depend on the fact that NGE creates a paced train by default.

---

Do we? I believe there was some back and forth when I was developing about whether we should have the fork overrule nge to create a trains schedule instead, and I think that got implemented in one version.